diff --git a/node_modules/@dcl/ui-env/dist/env.d.ts b/node_modules/@dcl/ui-env/dist/env.d.ts
index af8de20..acefb76 100644
--- a/node_modules/@dcl/ui-env/dist/env.d.ts
+++ b/node_modules/@dcl/ui-env/dist/env.d.ts
@@ -1,7 +1,8 @@
 export declare enum Env {
+    LOCAL = "local",
     DEVELOPMENT = "dev",
     STAGING = "stg",
-    PRODUCTION = "prod"
+    PRODUCTION = "prod",
 }
 /**
  * Returns an array with all the possible envs
diff --git a/node_modules/@dcl/ui-env/dist/location.js b/node_modules/@dcl/ui-env/dist/location.js
index da51e2b..70d7fa1 100644
--- a/node_modules/@dcl/ui-env/dist/location.js
+++ b/node_modules/@dcl/ui-env/dist/location.js
@@ -15,7 +15,7 @@ function getEnvFromTLD(location) {
     else if (host.endsWith('.today') || host.endsWith('.net')) {
         return env_1.Env.STAGING;
     }
-    else if (host.endsWith('.io') || host.endsWith('.zone')) {
+    else if (host.endsWith('.io') || host.endsWith('.zone') || host.endsWith('.vote')) {
         return env_1.Env.DEVELOPMENT;
     }
     return null;
